dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/maatapi/MaatApiWebClientFactory.java

```diff
@@ -52,6 +54,8 @@ public WebClient maatApiWebClient(
                     .resolver(DefaultAddressResolverGroup.INSTANCE)
                     .compress(true)
                     .responseTimeout(Duration.ofSeconds(90))
+                    .wiretap("reactor.netty.http.client.HttpClient",
+                        LogLevel.DEBUG, AdvancedByteBufFormat.TEXTUAL)
```

Comment:
Will this produce a lot of logging? Should it maybe be enabled by a configuration property?

Reply:
Yes, probably. I found it useful during debugging but it's not needed any more, so will remove it.